src/pool/srv_iv.c

```diff
@@ -276,6 +307,8 @@ pool_iv_prop_g2l(struct pool_iv_prop *iv_prop, daos_prop_t *prop)
 			D_FREE(owner_grp_alloc);
 		if (svc_list)
 			d_rank_list_free(dst_list);
+		if (pd_alloc)
+			D_FREE(pd_alloc);
```

Comment:
In general we prefer not to check for NULL before pd_alloc, it makes the code shorter/easier to read and the additional debug points are often useful.
```suggestion
		D_FREE(pd_alloc);
```